examples/slice/main.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"fmt"
+	"github.com/bitfield/script"
+	"log"
+)
+
+func main() {
+	// Read args, and return it in a Slice.
+	params, err := script.Args().Slice()
```

Comment:
Can we make this example program a bit more interesting? Ideally, each one would use `script` to do something potentially useful, or be a toy example of something useful.

Reply:
I tried to have a better example let me know if it sounds good to you.